wagtail/admin/templates/wagtailadmin/workflows/edit_task.html

```diff
@@ -41,14 +41,7 @@
                     <li class="actions footer__container">
                         {% block form_actions %}
                             <div class="dropdown dropup dropdown-button match-width">
-                                <button
-                                    type="submit"
-                                    class="button action-save button-longrunning"
-                                    data-controller="w-progress"
-                                    data-action="w-progress#activate"
-                                    data-w-progress-active-class="button-longrunning-active"
-                                    data-w-progress-active-value="{% trans 'Saving…' %}"
-                                >
+                                <button type="submit" class="button action-save button-longrunning" data-controller="w-progress" data-action="w-progress#activate" data-w-progress-active-value="{% trans 'Saving…' %}">
```

Comment:
Let's wrap these attributes to multiple lines - as the current approach does this.